src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -206,6 +291,97 @@
         az account management-group update --name GroupName --display-name DisplayName --parent ParentId/ParentName
 """
 
+helps['account management-group hierarchy-settings'] = """
+type: group
+short-summary: Provides operations for hierarchy settings defined at the management group level. Settings can only be set on the root Management Group of the hierarchy.
```

Comment:
Please use the first voice in the help description~
```suggestion
short-summary: Provide operations for hierarchy settings defined at the management group level. Settings can only be set on the root Management Group of the hierarchy.
```